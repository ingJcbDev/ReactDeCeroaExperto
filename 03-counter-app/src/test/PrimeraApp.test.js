import React from "react";
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';


import PrimeraApp from "../PrimerApp";

describe('Prueba PrimeraApp', () => {
//    test('Debe de mostrar "Hola soy goku"', () => {
//        const saludo = 'Hola soy goku';

//        const {getByText} = render(<PrimeraApp saludo={saludo}/>);
//        expect( getByText(saludo )).toBeInTheDocument();
//    });

test('Debe de mostrar primera prueba correctamente', () => {
    const saludo = 'Hola soy goku';
    const wrapper = shallow(<PrimeraApp saludo={saludo}/>);

    expect(wrapper).toMatchSnapshot();
})

});
