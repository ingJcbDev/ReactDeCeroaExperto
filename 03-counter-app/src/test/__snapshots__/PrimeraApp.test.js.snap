// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba PrimeraApp Debe de mostrar primera prueba correctamente 1`] = `
<Fragment>
  <h1>
    Hola soy goku
  </h1>
  <p>
    soy un subtitul
  </p>
</Fragment>
`;
